lib/iris/pandas.py

```diff
@@ -22,16 +24,18 @@
 try:
     from pandas.core.indexes.datetimes import DatetimeIndex  # pandas >=0.20
 except ImportError:
+    pass
```

Comment:
I'm not sure I understand the intention behind the `pass` statement. It seems like this code would work without it?

Reply:
Don't worry that was just me being a 🔧

9391f432ec987b3e6f948c1e5cca3ec679777e23